### DISCUSSION TITLE
Stopping injection after a specific time

### DISCUSSION THREAD

**Discussion by foojan-Ka** [GeoChemFoam/GeoChemFoam#41] [Q&A] [answered]
Hi,

Hope you are doing great.

My name is Foojan, and I wanted to ask a question about 2-phase flow in a micromodel (the second tutorial). I was wondering how can I stop flow rate after a specific time when running 2-phase flow, and then start it again after passage of specific time?

Looking forward to hearing from you.

Best,
Foojan

**Comment by GeoChemFoam** [comment, marked as answer]
Hi Foojan,

This is probably possible with some openfoam know how, but it is beyond what we are willing to support here. Please have a look/ask on the openfoam message boards.

Hannah.

From: Foojan-Ka ***@***.***>
Sent: Thursday, February 15, 2024 5:26 AM
To: GeoChemFoam/GeoChemFoam ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [GeoChemFoam/GeoChemFoam] Stopping injection after a specific time (Discussion #41)

****************************************************************
Caution: This email originated from a sender outside Heriot-Watt University.
Do not follow links or open attachments if you doubt the authenticity of the sender or the content.
****************************************************************


Hi,

Hope you are doing great.

My name is Foojan, and I wanted to ask a question about 2-phase flow in a micromodel (the second tutorial). I was wondering how can I stop flow rate after a specific time when running 2-phase flow, and then start it again after passage of specific time?

Looking forward to hearing from you.

Best,
Foojan

—
Reply to this email directly, view it on GitHub<https://github.com/GeoChemFoam/GeoChemFoam/discussions/41>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AWDOAFQPNBJMTZTMICACY7LYTWL7FAVCNFSM6AAAAABDJTRS6KVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGIZDAMRVGU>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.******@***.***>>

________________________________

Founded in 1821, Heriot-Watt is a leader in ideas and solutions. With campuses and students across the entire globe we span the world, delivering innovation and educational excellence in business, engineering, design and the physical, social and life sciences. This email is generated from the Heriot-Watt University Group, which includes:

  1.  Heriot-Watt University, a Scottish charity registered under number SC000278
  2.  Heriot- Watt Services Limited (Oriam), Scotland's national performance centre for sport. Heriot-Watt Services Limited is a private limited company registered is Scotland with registered number SC271030 and registered office at Research & Enterprise Services Heriot-Watt University, Riccarton, Edinburgh, EH14 4AS.

The contents (including any attachments) are confidential. If you are not the intended recipient of this e-mail, any disclosure, copying, distribution or use of its contents is strictly prohibited, and you should please notify the sender immediately and then delete it (including any attachments) from your system.
